### PR TITLE
New version SMP

### PR DESCRIPTION
**Pull request by NicolasRuth**
With short_version and new subscales

### CODE REVIEW

File: R/SMP.R [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
 #' @param label (Character scalar) Three uppercase letter acronym of the questionnaire.
 #' This is also the label given to the results in the output file.
+#'
 #' @param dict (i18n_dict) The psyquest dictionary used for internationalisation.
+#'
 #' @param subscales (Character vector) The subscales to be included in the questionnaire.
-#' Possible subscales are \code{"Reflective & Complex"}, \code{"Intense & Rebellious"},
-#' \code{"Upbeat & Conventional"}, and \code{"Energetic & Rhythmic"}.
-#' If no subscales are provided all subscales are selected.
+#' There are two possible sets subscales: 1. From the Do-re-mi paper: \code{"Reflective & Complex"},
```
Comment (fmhoeger) [inline R/SMP.R]: Rather "subscale sets"?

File: R/standalone.R [hunk 2]
```diff
@@ -459,20 +459,28 @@ SES_standalone <-
 #' The first language is selected by default.
 #'
 #' @param subscales (Character vector) The subscales to be included in the questionnaire.
-#' Possible subscales are \code{"Reflective & Complex"}, \code{"Intense & Rebellious"},
-#' \code{"Upbeat & Conventional"}, and \code{"Energetic & Rhythmic"}.
+#' There are two possible sets subscales: 1. From the Do-re-mi paper: \code{"Reflective & Complex"},
```
Comment (fmhoeger) [inline R/standalone.R]: Same, see above.

File: R/utils.R [hunk 3]
```diff
@@ -78,6 +78,15 @@ get_items <- function(label, subscales = c(), short_version = FALSE, configurati
 
       filtered_items <- as.data.frame(items[map(question_ids, function(x) grep(sprintf("TGMS_%04d", x), items$prompt_id)) %>% unlist() %>% unique(), ])
 
+      return(filtered_items[order(filtered_items$prompt_id), ])
+    } else if (label == "SMP") {
```
Comment (fmhoeger) [inline R/utils.R]: You are in the wrong `if else `branch I think. Maybe write a test for the short version which would have found this.

**Review by fmhoeger — CHANGES_REQUESTED** [review]

File: R/utils.R (hunk 3 above)
Comment (NicolasRuth) [reply R/utils.R]: Where else should it be? There are else if for GMS, SCS, SCA, too.  I can try a test.

**Comment by fmhoeger** [comment]
One more thing: Please run `devtools::check()` or press "Check" in the GUI in order to rebuild the `Rd` files.

**Comment by NicolasRuth** [comment]
rebuilding done.

**Pull request merged**